.yarnrc.yml

```diff
@@ -0,0 +1,10 @@
+nodeLinker: node-modules
+npmRegistryServer: https://registry.npmjs.org
+npmAuthToken: ${NPM_TOKEN}
+plugins:
+  - path: .yarn/plugins/@yarnpkg/plugin-interactive-tools.cjs
+    spec: "@yarnpkg/plugin-interactive-tools"
+  - path: .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs
+    spec: "@yarnpkg/plugin-workspace-tools"
+yarnPath: .yarn/releases/yarn-3.6.4.cjs
+npmPublishAccess: public
```

Comment:
Since this is a publicly available package, we need to specify it. By default, Yarn Berry creates private packages when you use a scope (e.g., `@ginger.io`)